docs-src/content/getting-started/installation.md

```diff
@@ -14,3 +14,11 @@ import this package into your application:
 ``` go
 import "github.com/cobaltspeech/sdk-cubic/grpc/go-cubic"
 ```
+
+### Python
+The Python SDK depends on Python >= 3.5. You may use pip to perform a system-wide install, or use virtualenv for a local install.
```

Comment:
pip or pip3?

Reply:
I think generally `pip` is soft linked to the appropriate version corresponding to the current environment's python version, so just `pip` should work